src/assets/index.js

```diff
@@ -0,0 +1,7 @@
+export const hero = require("./hero.jpg");
+export const headphone = require("./headphone.jpg");
+export const laptop = require("./laptop.jpg");
+export const mobile = require("./mobile.jpg");
+export const smartwatch = require("./smartwatch.jpg");
+export const speaker = require("./speaker.jpg");
+export const storage = require("./storage.jpg");
```

Comment:
you can use image source paths directly just like we do in HTML, for that your assets should be in `public` folder. 
Converting every single image in component with take lot of code and time

Reply:
Okay, I was doing like this for exporting all images from a single file.